proto/sharding/watchtower/watchtower.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
```

Comment:
Can we move this file to the proto/eth folder?

Reply:
inside v1alpha? or inside watchtower dir?

Reply:
moved